sale_purchase_customer/hooks.py

```diff
@@ -0,0 +1,9 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from psycopg2 import sql
+
+def pre_init_hook(cr):
```

Comment:
You could send an informative message to the log indicating that the field was created like here: https://github.com/OCA/account-financial-reporting/blob/b6fdadefe0744e9087b025225a704a454876ccfd/account_tax_balance/hooks.py#L15-L16